contracts/hyperlane-mailbox/tests/harness.rs

```diff
@@ -282,6 +313,159 @@ async fn test_latest_checkpoint() {
     assert_eq!(index, 0u32);
 }
 
+// ============ process ============
+
+#[tokio::test]
+async fn test_process_id() {
+    let (mailbox, ism_id, recipient_id, _) = get_contract_instance().await;
+
+    let metadata = vec![5u8; 100];
+
+    let agent_message = test_message(&mailbox, recipient_id.clone(), false);
+    let agent_message_id = agent_message.id();
+
+    let contract_inputs = vec![ism_id.clone(), recipient_id];
+
+    let process_call = mailbox
+        .methods()
+        .process(metadata.clone(), agent_message.clone().into())
+        .set_contract_ids(&contract_inputs)
+        .tx_params(TxParameters::default().set_gas_limit(1_200_000))
+        .call()
+        .await
+        .unwrap();
+
+    let message_id = &process_call.get_logs_with_type::<Bits256>().unwrap()[0];
```

Comment:
did you mean for this to be in https://github.com/hyperlane-xyz/fuel-contracts/pull/75 ?

Reply:
yeah this is gone now